Users/register

robert-stoian opened the pull request:
The implementation of the register form for users.

DenisaHalmaghi committed:
add delete action and refactor view

DenisaHalmaghi committed:
fix comparison issue

DenisaHalmaghi committed:
add edit functionality

DenisaHalmaghi committed:
add validators

DenisaHalmaghi committed:
handle null|empty values

DenisaHalmaghi committed:
display errors in views

DenisaHalmaghi committed:
remove unused class

DenisaHalmaghi committed:
apply post changes to edit controller

DenisaHalmaghi committed:
fix shit

robert-stoian committed:
add photo type

Created PhotoType enum, added a new field to the Photo entity: photoType.

robert-stoian committed:
Merge branch 'entity' into users/register

DenisaHalmaghi committed:
change form style

robert-stoian committed:
add User/Register User

DenisaHalmaghi committed:
add more fields to entity

DenisaHalmaghi committed:
change jobpost cards

LeUnicorn commented on JobRecruiting/JobRecruiting/src/main/java/com/recruit/jobrecruiting/user/servlet/AddUser.java:
```diff
@@ -0,0 +1,99 @@
+            fileContent = new byte[(int) fileSize];
+            filePart.getInputStream().read(fileContent);
+            userBean.addCv(id, fileName, fileType, fileContent);
+        } catch (NoSuchAlgorithmException ex) {
```
daca ai nevoie de mesaje de eroare si validatori te poti uital la jobPostValidator

robert-stoian replied:
Am prins exceptia aia acolo, pentru ca mai sus am folosit metoda creata de Andrei createUser, din UserBean, care arunca exceptia NoSuchAlgorithmException (cauzata de faptul ca parola e hashed cu sha256). Initial m-am gandit sa arunc si eu mai departe exceptia, fara sa o prind, dar imi da eroare, pt ca metoda doPost e overridden din HttpServlet.

robert-stoian replied:
Adaug imediat mesaje de eroare si validatori. Mersi :)

DenisaHalmaghi committed:
add filtering for jobs

DenisaHalmaghi committed:
add form template

DenisaHalmaghi committed:
refactor validator to template

DenisaHalmaghi committed:
add skill functionality

DenisaHalmaghi committed:
refactor job form

robert-stoian committed:
Merge branch 'jobpost' into users/register

DeeaMatei commented on JobRecruiting/JobRecruiting/src/main/java/com/recruit/jobrecruiting/ejb/JobPostBean.java:
```diff
@@ -0,0 +1,75 @@
+    }
+
+    public JobPostDetails getJobPost(int id) {
+        LOG.info("getCar");
```
Remove LOG

robert-stoian committed:
add validators

Added validators that check if the username is already taken, if the email is already used, if the password is secure, etc...

TheBobu committed:
Merge branch 'dev' into users/register
